src/pages/ImportWorkflow/ImportWorkflow.js

```diff
@@ -50,32 +55,68 @@ const styles = {
   },
 };
 
+const MethodSource = Object.freeze({
+  GitHub: 'GitHub',
+  Dockstore: 'Dockstore',
+});
+
 export const ImportWorkflow = ({ path, version, source }) => {
   const [isBusy, setIsBusy] = useState(false);
   const [confirmOverwriteInWorkspace, setConfirmOverwriteInWorkspace] = useState();
   const [workflowName, setWorkflowName] = useState(_.last(path.split('/')));
 
+  const signal = useCancellation();
   const { wdl, status: wdlStatus } = useDockstoreWdl({ path, version, isTool: source === 'dockstoretools' });
 
+  const importToAzureCromwellApp = async (workspaceId) => {
+    const appUrls = await Ajax(signal)
+      .Apps.listAppsV2(workspaceId)
```

Comment:
I'd recommend consuming the `Apps` module directly, instead of going through ajax. You'll find mocking app returns in tests a lot easier

Reply:
See CloudEnvironmentModal.ts
`import { Apps } from 'src/libs/ajax/leonardo/Apps';`

`Apps(signal)`

Reply:
Updated the code to directly call Apps module: https://github.com/DataBiosphere/terra-ui/pull/3964/files#diff-e4bce32c47ba1256103a5944f98474fd0893f6a3ead4cdd92ad628c59ecf3a8dR73-R75